server/client.go

```diff
@@ -477,7 +477,7 @@ func (c *client) readLoop() {
 		// Budget to spend in place flushing outbound data.
 		// Client will be checked on several fronts to see
 		// if applicable. Routes will never wait in place.
-		budget := time.Millisecond
+		budget := 500 * time.Microsecond
```

Comment:
Why the change here?

Reply:
Because this change was part of the fan-in fix. If I revert the fix, I want to restore the code as close as possible to what it was prior to the fix.

Reply:
ok gotcha.